### ISSUE TITLE
HTML Feedback

### ISSUE THREAD

**Issue by Enspired** [tshindollar/GDMA-2662-Assignment-3#1]
Index (header / footer comments apply to all pages) 
- [x] Don't include IDs on the body tag, use IDs where you may need them throughout the content https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L21 
- [x] Links to the current page should be '#' so that the page doesn't refresh. Alt text should communicate the same text as the image. https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L27
- [x] Extra div can be removed: https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L38 https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L39
- [x] 'sustainability starts with you' is one sentence, utilize a span and line break to stylize 
- [x] This is not the h1 to this page https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L41 
- [x] This is not a section, there is no intro copy, it should be a div https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L50 
- [x] .col-center won't do anything if the children total up to 12 https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L57 
- [x] Double check the .col- sizes https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L58 
- [x] .row.col-center only works with flex containers, the text can be controlled with text-align. Missing .wrapper around content  https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L79 https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L103
- [x] Extra div can be removed https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L103 
- [x] Remove any text that doesn't appear in browser and be sure to use the link from the prototype. Update phone and email to match the provided content https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L106 
- [x] Be sure to use the functions.js for smooth scroll https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/home.html#L118 

About 
- [x] Remove extra divs https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/about.html#L37  https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/about.html#L38
- [x]  Missing header around introductory content https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/about.html#L58 
- [x] Rather than a .row on the section, you'll have to split the UL into two separate ULs that will be in two columns, inside of a row. The .row only needs to be around the UL https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/about.html#L75 
- [x] This div can be removed https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/about.html#L71

Take Action 
- [x] Extra divs can be removed https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/take-action.html#L37 https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/take-action.html#L38
- [x] Not a section, has no introductory content. This div can go just around the first paragraph  https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/take-action.html#L45 
- [x] Missing .col- class to control the size https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/8401a2df270d1de60400b532008bbf98e147a45d/take-action.html#L46 
- [x] Hyphen in title can be removed. Be sure to update the title for each page examples: Bedford Foundation, About Bedford Foundation OR Home | Bedford Foundation, About | Bedford Foundation, etc.

**Comment by Enspired** [comment]
Index (header / footer comments apply to all pages) 
- [x] home page should be named index.html, when typing `www.website.com` it will look for the root file based on the name `index.html`. If the page were named _home.html_ you'd have to type `www.website.com/home.html ` for the url 
- [x] Id name can get more specific hero-home bg-mountain-top https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/home.html#L23 
- [x] Every page needs an h1 https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/home.html#L27 
- [x] Update alt text to communicate the same message as the written content in the image https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/home.html#L27 
- [x] You'll want this div just around the second paragraph since the second paragraph line wraps before the first https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/home.html#L45 
- [x] don't apply a width to text elements, include a div around the UL https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/home.html#L73 

About 
- [x] Include the IDs for the navigation to jump to https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/about.html#L50 
- [x] Include divs around the ULs to apply the width / .col- classes to https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/about.html#L73 

Take Action 
- [x] Extra divs can be removed https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/take-action.html#L37 https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/take-action.html#L38
- [x] Can include another div around this div for .row.col-center OR use margin: 0 auto; to center https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/take-action.html#L45 
- [x] Throw some # pound symbols in for the href value so they don't reload the page https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/9f48282467e8676515dc270b825e93bb0166505f/take-action.html#L55

**Comment by Enspired** [comment]
Index 
- [x] Missing the h1 on this page https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/index.html#L27 
- [x]  Should link to take action page https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/index.html#L66 
- [x] Missing a div above the header for the about and take action image. Consider if those images will need to be cropped as background images or if the full image needs to be visible at all times and will go in through the HTML  https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/index.html#L53 
- [x] Going to need a div outside of this .wrapper to apply the lighter green background color to https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/index.html#L73 
- [x] The social media IDs can be removed, apply margin to the LI's to create space between the icons https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/index.html#L82 

About 
- [x] not the h1 on this page, can only have one h1 per page https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/about.html#L25 
- [x] Move this .wrapper to a div https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/about.html#L109 
- [x] Remove social media IDs https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/about.html#L117 

Take Action 
- [x] Can wrap this content in another div with .row.col-center or use margin to center https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/take-action.html#L41 
- [x] Move .wrapper to a div https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/take-action.html#L75 
- [x] Remove social media IDs https://github.com/tshindollar/GDMA-2662-Assignment-3/blob/f9f413ec9fc5e7e706fd4a24de14f7cd0b069222/take-action.html#L83 

Images 
- [ ] Only images going through the HTML need retina counterparts
